lib/board.rb

```diff
@@ -0,0 +1,53 @@
+class Board
+  attr_reader :squares, :wining_combinations
+
+  def initialize
+    @squares = [
+      [{ '1': nil }, { '2': nil }, { '3': nil }],
+      [{ '4': nil }, { '5': nil }, { '6': nil }],
+      [{ '7': nil }, { '8': nil }, { '9': nil }]
+    ]
+
+    @wining_combinations = [
+      [1, 2, 3], [4, 5, 6], [7, 8, 9],
+      [1, 4, 7], [2, 5, 8], [3, 6, 9],
+      [1, 5, 9], [3, 5, 7]
+    ]
+  end
+
+  def show_board
+    puts '+---------------------------+'
+
+    @squares.each do |arr|
+      print '|'
+      arr.each do |el|
+        el.each do |key, value|
+          if value
+            print " #{key} - #{value == 'x' ? "\u{274C}" : "\u2B55"}  "
+          else
+            print " #{key} - \u{2754}  "
+          end
+        end
+      end
+      print '|'
+      puts
+    end
+
+    puts '+---------------------------+'
+  end
```

Comment:
## Don't use puts/print in a logic method
- No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
  - [ ] Create a return string variable
  - [ ] Assign to that variable the board and return

Example:

```ruby
def show_board
 my_board=''
...
my_board+= assign the board logic 
#return my_board variable
my_board
end

```